doc/tools/nordic_segger.rst

```diff
@@ -78,6 +78,38 @@ and ``<x>`` is either 1 for nRF51-based boards or 2 for nRF52-based boards
 
 Where ``<x>`` is either 1 for nRF51-based boards or 2 for nRF52-based boards
 
+Debugging with GDB
+******************
+
+**Note**: The instructions in this section are common to all Operating Systems.
+
+Once the application is flashed into the board, you can opt to debug using GDB
+or try a visual debugger such as Segger Ozone, descibed in :ref:`nordic_segger_ozone`.
+
+* Start the J-Link GDB server on one console:
```

Comment:
Better is to use numbers for steps, so use `#.` for the list items, and need to indent lines that are part of this list item.

The phrase ``a terminal window`` sounds better than ``one console`` here.